app/models/Gender.scala

```diff
@@ -18,8 +18,9 @@ package models
 
 object Gender {
   val list = List(
-    ("Male", false),
-    ("Female", false),
-    ("Other", true)
+    ("male", "Male", false),
```

Comment:
It *is* becoming a little bit tricky perhaps, but this style is used elsewhere and ui widgets are dependent on it. Will leave for now.